climada/util/api_client.py

```diff
@@ -486,3 +493,145 @@ def purge_cache(self, local_path):
         """
         dlf = Download.get(Download.path==str(local_path.absolute()))
         dlf.delete_instance()
+
+    def get_hazard(self, hazard_type=None, data_dir=SYSTEM_DIR):
```

Comment:
the functions gets_hazard get_exposures and get_litpop_default do very similar things, I think I could simplify them